examples/configs/model/ppo/continuous.yaml

```diff
@@ -1,4 +1,4 @@
-policy_wrapper: normal_param
+network_wrapper: normal_param
```

Comment:
I understand that we have 2 wrappers for the policy (one for the exploration and the other for the distribution), but network_wrapper seems too general to me: we have multiple networks usually, and several can have wrappers.
Why not `policy_net_wrapper` and `exploration_wrapper`?